packages/server/src/internals/cache/index.ts

```diff
@@ -44,7 +44,20 @@ export interface Cache {
 }
 
 export function createCache({ timer }: { timer: Timer }): Cache {
-  const client = new Redis(process.env.REDIS_URL)
+  const url = process.env.REDIS_URL.replace('redis://', '')
+  const client = new Redis({
+    host: url.split(':')[0],
+    port: Number(url.split(':')[1]),
```

Comment:
```suggestion
  const redisUrl = new URL(process.env.REDIS_URL)
  const client = new Redis({
    host: redisUrl.hostname,
    port: redisUrl.port,
```

Reply:
IMHO that's better to parse the REDIS url :smile:

Reply:
That's actually better, thanks @kulla

Reply:
```suggestion
  const redisUrl = new URL(process.env.REDIS_URL)
  const client = new Redis({
    host: redisUrl.hostname,
    port: Number(redisUrl.port),
```
(Casting the port to type Number has to stay.)